### ISSUE TITLE
Results tab shows Your file couldn’t be accessed

### ISSUE THREAD

**Issue by muh182** [openstudiocoalition/OpenStudioApplication#758]
Issue overview
--------------
Apologies if this issue is reported before. I was not able to find anything on the account. When the OpenStudio file name has a special character (e.g., `#`), the results tab does not show the results.

## Current Behavior
When the file name has the special character, the following error message is shown:

```
Your file couldn’t be accessedIt may have been moved, edited, or deleted.
ERR_FILE_NOT_FOUND
```

<img width="1277" alt="Screenshot 2024-10-04 at 5 16 40 PM" src="https://github.com/user-attachments/assets/deb6b9a7-6ed2-41c3-b226-a80d256675d4">

## Expected Behavior
The results table should show the EnergyPlus and OpenStudio results similar to the image below:
<img width="984" alt="Screenshot 2024-10-04 at 5 20 03 PM" src="https://github.com/user-attachments/assets/feb862ea-bdab-436a-963b-9516fee4d877">

## Steps to Reproduce
1. Just add `#` to any OpenStudio file name.

## Possible Solution
Remove `#` from the file name.

### Environment
I have seen this error before in different platforms and older version of OpenStudio but never reported it. The test is conducted on:

 - MAC 15.0.1 (24A348)
 - Version of OpenStudioApplication: 1.7

### Context
Recommended students to remove the `#` or see the reports folder.

Here is a [link](https://www.dropbox.com/scl/fi/tqq98711cq93s9uwor4po/Test-with-hashtag.zip?rlkey=lnttbe9avjma44uk4morh0gak&dl=0) to a sample model.

**Comment by macumber** [comment]
I confirmed the same behavior exists on Windows.  Probably just need to escape the filename path to make it valid url.  From http://www.faqs.org/rfcs/rfc1738.html: 

> The character "#" is unsafe and should always be encoded because it is used in World Wide Web and in other systems to delimit a URL from a fragment/anchor identifier that might follow it.

**Comment by jmarrec** [comment]
Fixed in #759, but as I say there:

> My two cents: NEVER USE special characters for files or directory names. Nor space.
> 
> Limit all file names to numbers ([0-9]) letters ([a-zA-Z]), and "-", "_" and "." You'll live a a much happier life generally.
